spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -560,10 +520,44 @@ public static class TestPojoWithAnnotatedArguments {
 }
 ----
 
-NOTE: Dispatching through `@StreamListener` conditions is only supported for handlers of individual messages, not for reactive programming support (described spring-cloud-stream-overview-reactive-programming-support[later]).
+*Content Type Negotiation in the context of `condition`*
+
+It is important to understand some of the mechanics behind content-based routing using `condition` argument of `@StreamListener`, especially in the context of type of the message as a whole.
+It may also help if you familiarise yourself with the <<Content Type Negotiation>>  before you proceed.
```

Comment:
Replace "familiarise" with "familiarize" (our standard is America spelling). Also, change <<Content Type Negotiation>> to "`<<Content Type Negotiation>>`" (quotation marks and backticks).